indico/modules/events/management/forms.py

```diff
@@ -215,7 +216,6 @@ def validate_contact_emails(self, field):
 
 
 class EventClassificationForm(IndicoForm):
-    keywords = IndicoTagListField(_('Keywords'))
```

Comment:
Maybe leave a comment here that a `keywords` field is added dynamically when creating the form. Otherwise someone may think `validate_keywords` below is useless.

Reply:
Commend added.